lib/net/hostlist.js

```diff
@@ -787,6 +787,24 @@ class HostList {
     return out;
   }
 
+  /**
+   * Convert address list to a filtered
+   * array. Only addresses without keys
+   * are included.
+   *
+   * @returns {NetAddress[]}
+   */
+
```

Comment:
I don't love the name of this method. Would like to rename it to something better.

Reply:
You could add a boolean to `toArray()` to filter and keep it as one function?

Reply:
`toStandardArray()` or `toStandardList()`?